Software/Source/PktLoss.py

```diff
@@ -0,0 +1,68 @@
+import json
+#Global Variable
+zero=0
+one=0
+two=0
+three=0
+four=0
+zero_los=0
+one_loss=0
+three_loss=0
+four_loss=0
+#count the total number of packet types (0,1,2,3 and 4) 
+def PacketCheck(check):
+    if(check=='0'):
+        global zero
+        zero=zero+1
+    elif(check=='1'):
+        global one
+        one=one+1
+    elif(check=='2'):
+        global two
+        two=two+1
+    elif(check=='3'):
+        global three
+        three=three+1
+    elif(check=='4'):
+        global four
+        four=four+1
+    return(zero,one,two,three,four)
+
+def losscal(): 
+    f=open("B0912265674C.txt","r")
+    while True:
+        content=f.readline()
+        if content:
+            data=json.loads(content)
+            check=data["PacketType"]
+            data1=PacketCheck(check)
+            #print(zero,one,two,three,four)
+        
+        else:
+            break

```

Comment:
Usage of break instruction, may not be logically correct in different scenarios. You may have some valid content in the next iterations of loop.